gatsby/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState, useCallback } from "react";
+import ChevronUp from "@nice-digital/icons/lib/ChevronUp";
+import throttle from "lodash/throttle";
+
+import styles from "./BackToTop.module.scss";
+
+const scrollThrottle = 100;
+const pagesScrollThreshold = 1.5; // number of viewports to scroll before "back to top" appears
+
+interface BackToTopProps {
+	scrollTargetId?: string;
+}
+
+export const BackToTop: React.FC<BackToTopProps> = ({
+	scrollTargetId: scrollTo = "content-start",
+}: BackToTopProps) => {
+	const [isFixed, setIsFixed] = useState(false);
+
+	const handleScroll = useCallback(() => {
+		const footer: Element | null = document.querySelector("footer");
+
+		if (!footer) return;
+
+		const footerVisible: boolean =
```

Comment:
Is this simpler?
```
const footerVisible: boolean = footer.getBoundingClientRect().top - window.innerHeight < 0
```

Reply:
(IE11 supports getBoundingClientRect().top)

Reply:
because IE11 doesn't support scrollY (just tested it). We could apparently use pageYOffset instead but the getBoundingClientRect solution seems nicer